modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/security/permission/resource/CPDisplayLayoutModelResourcePermission.java

```diff
@@ -0,0 +1,170 @@
+/**
```

Comment:
Revert this commit because we can directly use the model resource permission in the respective implementation of the Resource